typescript/cli/test-configs/zksync/chains/zksync2/metadata.yaml

```
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=../schema.json
+chainId: 270
+displayName: Zksync2
```

consistency with anvil metadata
```suggestion
displayName: zksync2
```